src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -1214,6 +1216,22 @@ def getMaxPacketSize(self):
         else:
             return 32
 
+    def setMaxPacketSize(self, packetSize):
+        """
+        Sets the maximum packet size of sensor.
+
+        packetSize (int): 32, 64, 128 and 256 are supported.
+        """
+
+        try:
+            packetSizes = {32: 0, 64: 1, 128: 2, 256: 3}
```

Comment:
We should also use this map for the `getMaxPacketSize` implementation to avoid two implementations (one with your map, another with the if-else) for consistency. Also I'm not sure if it is good to return `32` as a fallback in `getMaxPacketSize` - we should throw an exception instead because this behavior is not expected.

Reply:
Sure ;)